macros/blueprint-proc-macro/src/eigenlayer/event_listener.rs

```diff
@@ -0,0 +1,77 @@
+use proc_macro2::TokenStream;
+use quote::{format_ident, quote};
+use syn::{Ident, LitInt};
+
+use crate::job::EventHandlerArgs;
+
+pub(crate) fn generate_eigenlayer_event_handler(
+    fn_name_string: &str,
+    struct_name: &Ident,
+    _job_id: &LitInt,
+    event_handler: &EventHandlerArgs,
+    params_tokens: &[TokenStream],
+    _result_tokens: &[TokenStream],
+    additional_params: &[TokenStream],
+    fn_call: &TokenStream,
+) -> TokenStream {
+    let instance_base = event_handler.instance().unwrap();
+    let instance_name = format_ident!("{}Instance", instance_base);
+    let instance = quote! { #instance_base::#instance_name<T::T, T::P, T::N> };
+    let event = event_handler.event().unwrap();
+    let event_converter = event_handler.event_converter().unwrap();
+    let callback = event_handler.callback().unwrap();
+
+    quote! {
+        use alloy_network::Network;
+        use alloy_provider::Provider;
+        use alloy_transport::Transport;
+        use alloy_sol_types::SolEvent;
+        use alloy_sol_types::SolInterface;
+        use std::ops::Deref;
+        use gadget_sdk::events_watcher::evm::Config;
```

Comment:
This will not work, you need to use the absolute path everywhere. The main reason behind this, when this code expands it will contain these use stmts, However if we have different jobs, they will also expand and there duplication of the same thing.

Reply:
You're saying it won't work if these are imported already in the existing project, right?